core/code/chat.js

```diff
@@ -302,6 +308,30 @@ window.chat.nicknameClicked = function(event, nickname) {
 }
 
 window.chat.writeDataToHash = function(newData, storageHash, isPublicChannel, isOlderMsgs) {
+
+  if (newData.result.length > 0) {
+    //track oldest + newest timestamps/GUID
+    let first = {
```

Comment:
```suggestion
    var first = {
```

iitc uses ES5 - no `let`